### PR TITLE
refactor(@desktop/chat): make use of StatusQ's input in contact list …

### PR DESCRIPTION
**Pull request by PascalPrecht**
…search

Also, use `StatusPopupMenu` for starting new chats.

### CODE REVIEW

**Comment by PascalPrecht** [comment]
Screenshot 2021-07-26 at 15 35 38

File: ui/app/AppLayouts/Chat/CommunityComponents/CommunitiesPopup.qml [hunk 1]
```diff
@@ -44,6 +44,7 @@ StatusModal {
 
     content: Column {
         width: popup.width
+        property alias searchBox: searchBox
```
Comment (PascalPrecht) [inline ui/app/AppLayouts/Chat/CommunityComponents/CommunitiesPopup.qml:47]: These changes here are unrelated to the changes described in the PR and can potentially moved into a separate commit.

This is a fixing a bug that was introduced when community modals were refactored. Let me know if this should live in a separate commit.

File: ui/app/AppLayouts/Chat/ContactsColumn.qml [hunk 2]
```diff
@@ -11,26 +11,19 @@ import "./CommunityComponents"
...
     Layout.fillHeight: true
     width: 304
 
-    MouseArea {
-        anchors.fill: parent
-        onClicked: {
-            //steal focus from search field
-            addChat.forceActiveFocus();
-        }
-    }
```
Comment (jrainville) [inline ui/app/AppLayouts/Chat/ContactsColumn.qml:32]: I think we needed this to remove the focus from the search box

Comment (PascalPrecht) [reply ui/app/AppLayouts/Chat/ContactsColumn.qml:32]: Figured that, but it didn't seem to work. 
OTOH you also lose focus on the input when you click a chat item or the chat button or pretty much any other focussable component, so I personally doubt we need to keep this around.

Can add it back though if people experience a strong usability issue here

Comment (jrainville) [reply ui/app/AppLayouts/Chat/ContactsColumn.qml:32]: Ah I wonder if it got broken in master. I know it was @alexandraB99 who implemented this at first. I don't know who raised the issue first, but maybe @anastasiyaig would know. 
Context: do we care if the search box doesn't always lose focus (when clicking un-focusable stuff)

**Review by pinebit — APPROVED** [review]

**Comment by PascalPrecht** [comment]
This has landed as: https://github.com/status-im/status-desktop/commit/d47ddf69bb1e183d45a5806b4f805c9c85eed633
